Documentation/AndroidInstallation.md

```diff
@@ -31,29 +31,32 @@ dependencies {
 
 4.) In `android/app/src/main/java/com/xxx/MainApplication.java`
 
-After 0.19.0
+React Native versions 0.60.0 after
```

Comment:
Doesn't autolinking work with RN 0.60 ?

Reply:
With 0.60.0 onwards, React Native have breaking changes like migration to AndroidX. That means the Android native platform has ligth changes like the way to link the ```WebRTCModulePackage```